cirq-core/cirq/ops/clifford_gate_test.py

```diff
@@ -923,7 +923,39 @@ def test_all_single_qubit_clifford_unitaries():
     assert cirq.equal_up_to_global_phase(cs[23], (i - 1j * (-x - y - z)) / 2)
 
 
+def test_clifford_gate_repr():
+    q0, q1, q2 = cirq.LineQubit.range(3)
+    gates: list[cirq.ops.CliffordGate] = [
+        # Common gates
+        cirq.ops.CliffordGate.CNOT,
+        cirq.ops.CliffordGate.CZ,
+        cirq.ops.CliffordGate.SWAP,
+        # Other gates
+        cirq.ops.CliffordGate.from_op_list(
+            [cirq.ops.PhasedXZGate(axis_phase_exponent=0.25, x_exponent=-1, z_exponent=0).on(q0)],
+            [q0],
+        ),
+        cirq.ops.CliffordGate.from_op_list([cirq.ops.X(q0), cirq.CZ(q1, q2)], [q0, q1, q2]),
+    ]
+    for gate in gates:
+        t = gate.clifford_tableau
+        assert repr(gate) == f"Clifford Gate with Tableau:\n{t._str_full_()}"
```

Comment:
An assertion that literally repeats the tested code is not very useful as it in effect does `assert f() == f()`.
I suggest to remove it, the comparison with a literal string below is sufficient to catch the format bug fixed in this PR.

```suggestion
```

Reply:
Yeah, makes sense.